ZuluContent/Spells/Base/Spell.cs

```diff
@@ -304,7 +244,7 @@ public virtual bool CanCast()
 
         public virtual void SayMantra()
         {
-            if (IsWand() || m_SpellStrike)
```

Comment:
Why remove the spellstrike check?

Reply:
I originally implemented `m_SpellStrike` as a way to bypass the casting process because there was no common way execute just the spell effect independently. Now with `ITargetableAsync` and `IAsync` spell exist we can just directly trigger the spell effect which makes the spellstrike bool obsolete.

Reply:
ok cool